themes/_core/js/product.js

```diff
@@ -311,4 +311,17 @@ $(document).ready(() => {
             }
         });
     });
+
+    prestashop.on('updatedProduct', (args) => {
+        if (!args.product_url) {
```

Comment:
Don't you want to check `args.id_product_attribute` as well?

Reply:
Done! Thanks

Reply:
@mickaelandrieu One more time, double review was needed :trollface: